config/locales/en.yml

```diff
@@ -7,8 +7,13 @@ en:
       credit_balance: Store Credit Balance
     user_mailer:
       reset_password_instructions:
-        instructions_1: "A request to reset your password has been made.\nIf you did not make this request, simply ignore this email.\n\nIf you did make this request just click the link below:"
-        instructions_2: "If the above URL does not work try copying and pasting it into your browser.\nIf you continue to have problems please feel free to contact us."
+        welcome: Hi %{email}
+        instructions_1: A request to reset the password for your %{store_name} account has been made.
+        instructions_2: Click the button below to reset it.
+        instructions_3: If you did not request a password change ignore this email or contact us.
+        button: Reset your password
+        thanks: Thank you,
+        store_team: "%{store_name} Team"
```

Comment:
maybe you should ommit this interpolation like two above to keep consistency 😉

Reply:
Without this interpolation I'm getting Syntax Error:
```
<Psych::SyntaxError: (.../spree_auth_devise/config/locales/en.yml): found character that cannot start any token while scanning for the next token at line 16 column 21>
```

Reply:
oh, here: https://github.com/spree/spree_auth_devise/pull/504/files#diff-37c9b835927e3c363588f172de9bb6faR10 working without interpolation 🤔

Reply:
Seems like `%` char cannot be at the beginning of the phrase without interpolation

Reply:
so, maybe we should stick to rubocop style guide: https://rubystyle.guide/#string-interpolation ?